result/isislsp.yaml

```diff
@@ -0,0 +1,352 @@
+          description: |-
+            Up (0)-used when a prefix is initially advertised within the ISIS L3 hierarchy, 
+              and for all other prefixes in L1 and L2 LSPs. (default) 
+            Down (1)-used when an L1/L2 router advertises L2 prefixes in L1 LSPs. 
+            The prefixes are being advertised from a higher level (L2) down to a lower level (L1). 
+          type: string
+          enum: [up, down]
+        default_metric:
+          description: |-
+            ISIS default metric value.
+          type: integer
+          minimum: 0
+          maximum: 63
+        origin_type:
+          description: >-
+            The origin of the advertised route-internal or external to the ISIS area.
+            Options include the following: 
+              Internal-for intra-area routes, through Level 1 LSPs. 
+              External-for inter-area routes redistributed within L1, through Level 1 LSPs.
+          type: string
+          enum: [internal, external]
+  
+    IsisLsp.ExtendedIpv4Reachability:
+      description: |-
+        This container defines list of IPv4 extended reachability information.
+      type: object
+      properties:
+        prefixes:
+          description: >-
+            IPv4 prefix contained within extended reachability TLVs.
+          type: array
+          items:
+            $ref: '#/components/schemas/IsisLsp.ExtendedV4Prefix'
+
+    IsisLsp.ExtendedV4Prefix:
+      description: >-
+        This group defines attributes of an IPv4 standard prefix.
+      type: object
+      properties:
+        ipv4_address:
+          description: >-
+            An IPv4 unicast address
+          type: string
+          format: ipv4
+        metric:
+          description: >-
+            ISIS wide metric.
+          type: integer
+        redistribution_type:
+          description: |-
+            Up (0)-used when a prefix is initially advertised within the ISIS L3 hierarchy, 
+              and for all other prefixes in L1 and L2 LSPs. (default) 
+            Down (1)-used when an L1/L2 router advertises L2 prefixes in L1 LSPs. 
+            The prefixes are being advertised from a higher level (L2) down to a lower level (L1). 
+          type: string
+          enum: [up, down]
+        prefix_attributes:
+          $ref: '#/components/schemas/IsisLsp.PrefixAttributes'
+        
+
+    IsisLsp.Ipv6Reachability:
+      description: |-
+        This container defines list of IPv6 extended reachability information.
+      type: object
+      properties:
+        prefixes:
+          description: >-
+            IPv6 prefix contained within reachability TLVs.
+          type: array
+          items:
+            $ref: '#/components/schemas/IsisLsp.V6Prefix'
+
+    IsisLsp.V6Prefix:
+      description: >-
+        This group defines attributes of an IPv6 standard prefix.
+      type: object
+      properties:
+        ipv6_address:
+          description: >-
+            An IPv6 address
```

Comment:
How is the mask made available for a prefix ( also applicable to gnmi )

Reply:
will correct there too.